packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -1195,11 +1206,11 @@ describe("Garbage Collection Tests", () => {
             assert(deletedNodesBlob.type === SummaryType.Handle, "Deleted nodes state should be a handle");
 
             const refreshSummaryResult: RefreshSummaryResult = { latestSummaryUpdated: true, wasSummaryTracked: true, summaryRefSeq: 0 };
-			await garbageCollector.refreshLatestSummary(
```

Comment:
Look! tabs 🤣

Reply:
Haha I know.  Must have missed them on my last PR in this branch